IPython/kernel/zmq/kernelbase.py

```diff
@@ -699,3 +701,14 @@ def _at_shutdown(self):
             self.session.send(self.iopub_socket, self._shutdown_message, ident=self._topic('shutdown'))
             self.log.debug("%s", self._shutdown_message)
         [ s.flush(zmq.POLLOUT) for s in self.shell_streams ]
+
+    def _flush_execute_output(self):
```

Comment:
Naming point: 'execute output' implies something a bit different. I would call this `_flush_stdstreams`.